Broken example of how to open a modal in the documentation

### Description

I tried following the examples [given in the example documentation](https://slack.dev/node-slack-sdk/web-api#opening-modals) to open a modal, and it did not work and produced an error `invalid_arguments`

After digging a bit, the actual error explanation returned was 

> [ERROR] must define `submit` to use `input` block [json-pointer:/view/blocks/0/input]

It seems opening a modal via `web.views.open(openModalPayload)` requires the payload to have a submit field to work and will crash otherwise

This should be updated in the documentation (especially the "getting started" examples

### What type of issue is this? (place an `x` in one of the `[ ]`)
- [ ] bug
- [ ] enhancement (feature request)
- [ ] question
- [x] documentation related
- [ ] testing related
- [ ] discussion

### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

---

### Bug Report

Filling out the following details about bugs will help us solve your issue sooner.

#### Packages:

Select all that apply:

- [x] `@slack/web-api`
- [ ] `@slack/events-api`
- [ ] `@slack/interactive-messages`
- [ ] `@slack/rtm-api`
- [ ] `@slack/webhooks`
- [ ] I don't know

#### Reproducible in:

package version: @slack/web-api@^5.1.0", "@slack/web-api@^5.2.0"

node version: v11.5.0

#### Steps to reproduce:

1. Start a nodejs app
2. Follow the example given in the documentation to open a modal and copy the code to your own app
3. Run the code up till this point

#### Expected result:

Expected the example to work

#### Actual result / Attachment

```
{ Error: An API error occurred: invalid_arguments
    at Object.platformErrorFromResult (/.../node_modules/@slack/web-api/src/errors.ts:94:5)
    at WebClient.apiCall (/.../node_modules/@slack/web-api/src/WebClient.ts:159:13)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
  code: 'slack_webapi_platform_error',
  data:
   { ok: false,
     error: 'invalid_arguments',
     response_metadata: { messages: [Array], scopes: [Array] } } }
{ messages:
   [ '[ERROR] must define `submit` to use `input` block [json-pointer:/view/blocks/0/input]' ],
```

--- a/docs/_packages/web_api.md
+++ b/docs/_packages/web_api.md
@@ -268,6 +268,10 @@ const trigger = 'VALID_TRIGGER_ID';
         type: 'plain_text',
         text: 'Modal title'
       },
+      submit: {
+        type: 'plain_text',
+        text: 'Submit'
+      },
       blocks: [
         {
           type: 'input',

